sections/conclusion/conclusion-chapter.tex

```diff
@@ -20,6 +20,9 @@ \subsection{Future work}
 \\
 Additionally, we have experienced that not all contexts are equal in terms of importance.
 Due to this, it could be beneficial to include an attention mechanism that is able to learn the importance of each context or context combination for the convolution layer.
+This kind of attention idea could also be applied to layers.
+As we increase the amount of layers in a GCN, more noise will be introduced to the convolutions.
+To counter this, it may be beneficial to look into more advanced weighting mechanisms for layers, such that nodes further away from the root node are less influencial in the layer combination operation.
```

Comment:
```suggestion
To counter this, it may be beneficial to look into more advanced weighting mechanisms for layers, such that nodes further away from the root node are less influential in the layer combination operation.
```